skypeweb/skypeweb_login.c

```diff
@@ -167,10 +169,12 @@ skypeweb_login_got_t(PurpleUtilFetchUrlData *url_data, gpointer user_data, const
 			gchar *session_state = skypeweb_string_get_chunk(url_text, len, ":'https://login.live.com/GetSessionState.srf?", "',");
 			if (session_state) {
 				//These two appear to have different object keys each request :(
+				/*

```

Comment:
are side effects of this block calling into skypeweb_string_get_chunk() somehow used, and I don't see them? Or is this dead code that could be dropped altogether?